atlasdb-dbkvs-hikari/src/test/java/com/palantir/nexus/db/pool/config/OracleConnectionConfigTest.java

```diff
@@ -163,4 +166,23 @@ public void tcpsSetsTrustStore() throws IOException {
                 .port(PORT);
     }
 
+    @Test
+    public void protocolCaseInsensitiveTest() throws IOException {
```

Comment:
Would you mind writing a test that verifies that this does throw if an invalid protocol is passed?

Reply:
Done.